btf_interp/src/lib.rs

```diff
@@ -16,42 +17,42 @@ pub struct VirtualMachine<T> {
 
 impl<T> VirtualMachine<T> {
     /// Create VM based on the size, by default is 30 000. Also, it can be adjusted, by default it doesn't.
-    pub fn new(size: Option<usize>, adjust_tape: Option<bool>) -> Self {
+    pub fn new(size: Option<NonZeroUsize>, adjust_tape: Option<bool>) -> Self {
         VirtualMachine {
             tape: Vec::new(),
-            tape_size: size.unwrap_or(30000),
+            tape_size: size.unwrap_or(NonZeroUsize::new(3000).unwrap()),
```

Comment:
```suggestion
            tape_size: size.map(NonZeroUsize::get).unwrap_or(3000),
```

Neater and fewer unwraps 😁